lib/driver.js

```diff
@@ -100,6 +100,7 @@ class XCUITestDriver extends BaseDriver {
       let { device, udid, realDevice } = await this.determineDevice();
       log.info(`Determining device to run tests on: udid: '${udid}', real device: ${realDevice}`);
       this.device = device;
+      this.opts.device = device;
       this.opts.udid = udid;
       this.opts.isRealDevice = realDevice;
```

Comment:
I thing `this.opts.isRealDevice` should be `this.realDevice`. The `is...` properties are generally functions to check a condition throughout Appium. Plus in the next file the name is changed.